code/API_definitions/CAMARA Connectivity Insights API.yaml

```diff
@@ -1,46 +1,69 @@
 ---
-    openapi: 3.0.3
+    openapi: 4.0.0
```

Comment:
Why 4.0.0? ... CAMARA is using 3.0.3, see Design Guidelines.
BTW: 4.0.0 is not even released (see https://github.com/OAI/sig-moonwalk).